rockcraft.yaml

```diff
@@ -0,0 +1,63 @@
+name: openfga
+base: bare
+build-base: ubuntu:22.04
+version: "1.3.3"
+summary: Openfga Authorization Server
+description: |
+  OpenFGA is a flexible Authorization system inspired by Google's Zanzibar, designed for reliability and low latency at scale.
+license: Apache-2.0
+run-user: _daemon_
+platforms:
+  amd64:
+
+services:
+  openfga:
+    override: replace
+    command: openfga run
+    startup: disabled
+checks:
+  up:
+    override: replace
+    level: alive
+    exec:
+      command: grpc_health_probe -addr localhost:8081
```

Comment:
Do we want to have the healthcheck in the rockcraft file? I think that it makes sense because we are installing the `grpc_health_probe`.

Reply:
sounds good to me